src/condor_tools/htcondor_cli/annex.py

```diff
@@ -6,6 +6,129 @@
 from htcondor_cli.verb import Verb
 
 
+def increment(hash, key, subkey, subsubkey, number):
+    value = hash[key][subkey].get(subsubkey, 0)
+    value += number
+    hash[key][subkey][subsubkey] = value
+
+
+class Status(Verb):
+    """
+    Summarize the usage of an annex.
+    """
+
+    options = {
+        "annex_name": {
+            "args": ("--annex-name",),
+            "help": "annex name",
+        },
+    }
```

Comment:
```suggestion
    options = {
        "annex_name": {
            "args": ("--annex-name",),
            "help": "annex name",
        },
        "user": {
            "args": ("--user",),
            "help": "Display status for user. Defaults to current user (%(default)s)",
            "default": getpass.getuser(),
        },
    }
```
Per `Schedd.query()` comment. Still need to add `import getpass`, which GitHub won't let me suggest changes at the beginning of the script...

Reply:
I think/hope adding a `--user` flag to `htcondor job` is out-of-scope.  (I'm not sure that the "simplified" interface should have such an option.)  We'll just use `getpass.getuser()` -- to be consistent with other places we get the username, rather than out of any expectation that it'll consistently be the ID that the user authenticates to HTCondor as -- for the constraint, unless we can find a better way (e.g., make condor do it for us).